### ISSUE TITLE
Pylint complains functions in torch.linalg not callable.

### ISSUE THREAD

**Issue by hzhangxyz** [pylint-dev/pylint#9218]
### Bug description

```python
"""
Test pytorch linalg module with pylint
"""

import torch


def my_qr_decomposition(tensor: torch.Tensor) -> tuple[torch.Tensor, torch.Tensor]:
    """
    My QR
    """
    tensor_q, tensor_r = torch.linalg.qr(tensor)
    return tensor_q, tensor_r
```

### Command used

```shell
python -m pylint main.py "--generated-member=torch.*,torch.linalg.*"
```

### Pylint output

```python
************* Module main
main.py:12:25: E1102: torch.linalg.qr is not callable (not-callable)

-----------------------------------
Your code has been rated at 0.00/10
```

### Expected behavior

It is expected that pylint knows torch.linalg.qr is callable

### Pylint version

```shell
pylint 3.0.2
astroid 3.0.1
Python 3.11.5 (main, Aug 28 2023, 12:39:56) [GCC 11.4.0]
```

### OS / Environment

github action, see details at https://github.com/hzhangxyz/test-pytorch-linalg-pylint/actions/runs/6761431449/job/18376293735 .

Note: pylint 2.17.7 works well in this examples.

### Additional dependencies

```python
pytorch==2.0.1+cpu
```

**Comment by enrico-stauss** [comment]
I'm experiencing the same with some of the torch functionals (`torch.nn.functional.(pad|conv2d)`).

**Comment by Pierre-Sassoulas** [comment]
More related torch issues in https://github.com/pylint-dev/pylint/issues/10710 is someone work on this.
